resources/lang/en/beatmaps.php

```diff
@@ -191,6 +192,11 @@
                 'relevance' => 'relevance',
                 'nominations' => 'nominations',
             ],
+            'supporter_filter_quote' => [
+                'before_link' => 'Filtering by :filters requires an active ',
+                'link_text' => 'supporter tag',
+                'after_link' => '',
```

Comment:
crowdin doesn't allow translation of empty text so have fun using that `dangerouslySetInnerHTML`.